.github/workflows/pipeline.yml

```
@@ -34,9 +51,9 @@ jobs:
   coverage:
     needs: [ lint ]
     runs-on: ubuntu-24.04
+    
+    image: fcsproj/fastcalosim-alma9:${{ (github.event_name == 'pull_request' && needs.docker-check.outputs.docker_changed == 'true') && format('pr-{0}', github.event.pull_request.number) || 'latest' }}
```

actionlint (1.7.4)

unexpected key "image" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)